portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,17 @@
 // limitations under the License.
 
 
-fetch('/data').then(response => response.json()).then((comment) => {
-	let doc = document.getElementById('comment-container');
-	for(x in comment){
-		doc.innerHTML += "<li>";
-		doc.innerHTML += "<h3>" + comment[x].name + "</h3>";
-		doc.innerHTML += "<p>" + comment[x].content + "</p>";
-		doc.innerHTML += "</li>";
+fetch('/data').then(response => response.json()).then((comments) => {
+	let commentSection = document.getElementById('comments-container');
+    
+	for(x in comments){
+
+        commentSection.innerHTML += addComment(comments[x]);
 	}
 });
```

Comment:
Repeating a comment from #3:

It would be good to wrap this up in a function, even if we just immediately invoke it from this file or in some place from the HTML. Most of the JavaScript you'll write will be in a function, which gives us more control over when it runs and makes it reusable. Right now, it runs sometime after the script is loaded by the browser... but it could happen any time, which isn't always what we want.